New Code for 1.R

```diff
@@ -0,0 +1,101 @@
+#Load Libraries
+library(stringr)
```

Comment:
Is this the version you want me to grade? The point of version control is to keep the same code in the same place, so you want to avoid having extra files like this. They make it confusing for people reading your code who aren't on your team.